### PR TITLE
Prettier and other setup

### PR DESCRIPTION
**Pull request by jacastanon01**
# What was done
To ensure everyone is using the same coding standards and syntax, I added a settings.json to the project root. Make sure to have the prettier and eslint extensions installed in vscode. My hope is this will keep our code consistent and minimize unnecessary diffs in git (for example, changing of quotes, semi-colons or tab spaces) 

I also installed a package to organize tailwind classnames in accordance to their documentation and opted to use es6 import modules in the backend to stay consistent with these standards and removed some boilerplate files. I moved the products array into its own constants file to clean up `backend/index.js` so we can have dummy data in that file for testing purposes.  

## Additional info
May need to verify that the eslint and prettier configs are working correctly in your remote workspace

### CODE REVIEW

File: frontend/src/reportWebVitals.js
Comment (hinchley2018) [inline frontend/src/reportWebVitals.js:1]: please add this back

File: frontend/src/index.js [hunk 1]
```diff
@@ -2,16 +2,10 @@ import React from 'react';
...
 root.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>
 );
-
-// If you want to start measuring performance in your app, pass a function
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
```
Comment (hinchley2018) [inline frontend/src/index.js]: please add these changes back

File: backend/index.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 // Import the Express module
-const express = require('express');
+import express from 'express';
+import { products } from './constants.js';
```
Comment (hinchley2018) [inline backend/index.js:5]: this is un-needed we are going to just pull from mongo, I don't want un-needed files in the project cuz we will forget to delete.

**Review by hinchley2018 — CHANGES_REQUESTED** [review]
See requested changes LMK if you have questions :)

**Comment by hinchley2018** [comment]
Also there are some conflicts so if you merge `origin/main` into your branch it should resolve

File: backend/index.js (hunk 2 above)
Comment (jacastanon01) [reply backend/index.js:5]: Could this be removed once mongo is successfully integrated? Someone may be using the dummy data for some frontend issues

**Review by hinchley2018 — APPROVED** [review]

**Pull request merged**